example/main.go

```diff
@@ -2,14 +2,31 @@ package main
 
 import (
 	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
 
-	"github.com/getlantern/pacon"
+	"github.com/getlantern/pac"
 )
 
 func main() {
-	pacon.PacOn("localhost:12345/pac")
-	fmt.Println("proxy set, any key to continue...")
+	pac.SetHelperPath(absPath("./pac-cmd"))
+	if runtime.GOOS == "darwin" {
+		pac.SetIconPathOnOSX(absPath("icon.png"))
```

Comment:
I don't expect the icon and prompt to be set at multiple place. Not setting them doesn't affect the function. As it only makes sense on Mac, and is unrelated to the actual operation of setting pac, pass it in pac.On() and pac.Off() is a bit odd. I'll document the usage.